README.md

```diff
@@ -70,8 +93,16 @@ Before pushing your commit, ensure that the application works okay in production
 yarn cross-env DEBUG_PROD=true yarn start
 ```
 
+## Additional Resources
+
+- Amazon Chime SDK Classroom demo uses electron builder to ship the application for Windows and MacOS. Refer to multi platform build documentation as it is a great resource to help debug packaging failures: [Electron Multi Platform Build](https://www.electron.build/multi-platform-build)
+
 ## Troubleshooting
 
 ### I get "The application ... can't be opened" when opening the app
 
-The default zipping tool on MacOS Catalina may incorrectly unzip the package. Download an alternative (such as "The Unarchiver"), and unzip the package by right clicking and selecting "Open as". You may also need to adjust your security & privacy settings if you get an "unidentified developer" message.
\ No newline at end of file
+The default zipping tool on MacOS Catalina may incorrectly unzip the package. Download an alternative (such as "The Unarchiver"), and unzip the package by right clicking and selecting "Open as". You may also need to adjust your security & privacy settings if you get an "unidentified developer" message.
+
+### resize2fs: Bad magic number in super-block while trying to open /dev/xvda1
+
+You might get this error from the `cloud9-resize.sh` script while executing option 1. This error is thrown by the deploy script but the volume size is successfully updated. A consecutive run of the deploy script should pass successfully.
```

Comment:
nit: instead of "executing option 1" use "if using cloud9 to deploy"?

Reply:
Done